### PR TITLE
Add missing sidebars

### PR DESCRIPTION
**Pull request by OnkarRuikar**

### CODE REVIEW

File: files/en-us/web/api/beforeinstallpromptevent/prompt/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ tags:
 browser-compat: api.BeforeInstallPromptEvent.prompt
 ---
 
-{{SeeCompatTable}}
+{{QuicklinksWithSubPages("Web/API/BeforeInstallPromptEvent")}}{{SeeCompatTable}}
```
Comment (wbamberg) [inline files/en-us/web/api/beforeinstallpromptevent/prompt/index.md:15]: This ought to be APIRef I think. All sidebars under Web/API should either be DefaultAPISidebar or APIRef.

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thank you @OnkarRuikar !

File: files/en-us/web/api/beforeinstallpromptevent/prompt/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/api/beforeinstallpromptevent/prompt/index.md]: I see. Used `APIRef`.

**Review by wbamberg — APPROVED** [review]
👍 thank you!

**Pull request merged**